tcms/rpc/api/forms/testrun.py

```diff
@@ -68,7 +68,10 @@ def clean_status(self):
         return data
 
 
-class UpdateExecutionForm(BaseCaseRunForm):
```

Comment:
The base class is renamed in #1796 but it will no longer be in use after this change so it needs to be removed.